src/components/photoPlayer/plugin.js

```diff
@@ -1,23 +1,18 @@
-define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackManager', 'appRouter', 'appSettings', 'connectionManager'], function (browser, require, events, appHost, loading, dom, playbackManager, appRouter, appSettings, connectionManager) {
-    'use strict';
-
-    function PhotoPlayer() {
-
-        var self = this;
-
-        self.name = 'Photo Player';
-        self.type = 'mediaplayer';
-        self.id = 'photoplayer';
-
-        // Let any players created by plugins take priority
-        self.priority = 1;
+import connectionManager from 'connectionManager';
+
+export class PhotoPlayer {
```

Comment:
Should we specify this as the `default` export?

Reply:
I used the same format we seem to be using everywhere else, which is adding it to the bottom of the file.

Reply:
That seems odd. Why not just do `export default class PhotoPlayer`?

Reply:
@thornbill I can add that in a separate pull request if that's the only other reservation you had about the changes here.

Reply:
I just checked and this is actually the only instance, just removed it now.

Reply:
> I just checked and this is actually the only instance, just removed it now.

You are not the only one who has exported `default` at the end of file: #1089 (`backdropScreensaver`)

Reply:
Ah, well we haven't merged that yet so there's no problem quite yet.